qipr_approver/approver/parsers/mesh_ascii/actions.py

```diff
@@ -0,0 +1,154 @@
+    'DX': lambda acc, RHS: set_date(acc, 'major_descriptor_date', RHS),
+    # 'EC': lambda acc, RHS: set_return(acc, 'annotation', RHS),
+    'ENTRY': lambda acc, RHS: add_return(acc, 'entry', RHS, entry_lookup),
+    'FR': lambda acc, RHS: set_return(acc, 'frequency', RHS),
+    # 'FX': lambda acc, RHS: add_return(acc, 'forward_reference', RHS, desc_mh_lookup),
+    # 'HM': lambda acc, RHS: add_return(acc, 'heading_mapped_to', RHS, desc_mh_lookup),
+    'HN': lambda acc, RHS: set_return(acc, 'history_note', RHS),
+    # 'II': lambda acc, RHS: add_return(acc, 'indexing_information', RHS, desc_mh_lookup),
+    'MH': lambda acc, RHS: set_return(acc, 'mesh_heading', RHS),
+    # 'MH_TH': lambda acc, RHS: set_return(acc, 'mesh_heading_thesaurus', RHS),
+    'MN': lambda acc, RHS: set_return(acc, 'mesh_tree_number', RHS, mesh_num_lookup),
+    'MR': lambda acc, RHS: set_date(acc, 'major_revision_date', RHS),
+    'MS': lambda acc, RHS: set_return(acc, 'mesh_scope_note', RHS),
+    # 'N1': lambda acc, RHS: set_return(acc, 'castype1_name', RHS),
+    'NM': lambda acc, RHS: set_return(acc, 'substance_name', RHS),
+    'NM_TH': lambda acc, RHS: set_return(acc, 'substance_name_term_thesaurus', RHS),
+    'NO': lambda acc, RHS: set_return(acc, 'note', RHS),
+    # 'OL': lambda acc, RHS: set_return(acc, 'online_note', RHS),
+    'PA': lambda acc, RHS: add_return(acc, 'pharmacological_action', RHS, pharma_name_lookup),
+    # 'PI': lambda acc, RHS: set_return(acc, 'previous_indexing', RHS),
+    # 'PM': lambda acc, RHS: set_return(acc, 'public_mesh_note', RHS),
+    # this needs to be parsed out. complex data right now. Dont understand it
+    'PRINT ENTRY': lambda acc, RHS: add_return(acc, 'entry', RHS, entry_lookup),
+    'QA': lambda acc, RHS: set_return(acc, 'abbreviation', RHS),
+    # 'QE': lambda acc, RHS: set_return(acc, 'qualifier_entry_version', RHS),
+    # 'QS': lambda acc, RHS: set_return(acc, 'qualifier_sort_version', RHS),
+    # 'QT': lambda acc, RHS: set_return(acc, 'qualifier_type', RHS),
+    # cross references need more help
+    # 'QX': lambda acc, RHS: set_return(acc, 'qualifier_cross_reference', RHS),
+    'RECTYPE': lambda acc, RHS: instantiate_mesh_model(acc, RHS),
+    # 'RH': lambda acc, RHS: set_return(acc, 'running_head', RHS),
+    'RN': lambda acc, RHS: set_return(acc, 'cas_registry_number', RHS),
+    # multiples
+    'RR': lambda acc, RHS: add_return(acc, 'related_registry_number', RHS, reg_num_lookup),
+    'SH': lambda acc, RHS: set_return(acc, 'sub_heading', RHS),
+    'SO': lambda acc, RHS: add_return(acc, 'source', RHS, source_lookup),
+    'ST': lambda acc, RHS: add_return(acc, 'semantic_type', RHS, semantic_lookup),
+    'SY': lambda acc, RHS: add_return(acc, 'synonym', RHS, synonym_lookup),
+    # 'TH': lambda acc, RHS: set_return(acc, 'thesaurus_identifier', RHS),
+    # 'TN': lambda acc, RHS: set_return(acc, 'tree_node_allowed', RHS),
+    'UI': lambda acc, RHS: set_return(acc, 'ui', RHS),
+}
```

Comment:
I'll document why I skipped those parts of parsing. They didn't provide much value so I avoided using them the first time around but we may want to bring in that information later.